CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## 3.20.2 (Sep 13, 2023)
+IMPROVEMENTS:
+* ` to ensure that the unit of time being used is specified ([#2015](https://github.com/hashicorp/terraform-provider-vault/pull/2015))
```

Comment:
we don't need any changelog updates for documentation changes :)

Reply:
Removed Changelog changes, thank you for the feedback.